sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -812,6 +812,75 @@ class MapboxMap internal constructor(
     }
   }
 
+  /**
+   * Returns all the leaves (original points) of a cluster (given its cluster_id) from a GeoJsonSource, with pagination support: limit is the number of leaves
+   * to return (set to Infinity for all points), and offset is the amount of points to skip (for pagination).
+   *
+   * Requires configuring the source as a cluster by calling [GeoJsonSource.Builder#cluster(boolean)].
+   *
+   * @param sourceIdentifier GeoJsonSource identifier.
+   * @param cluster Cluster from which to retrieve leaves from
+   * @param limit The number of points to return from the query (must use type 'uint64_t', set to maximum for all points). Defaults to 10.
```

Comment:
I see that `uint64_t` is used everywhere to describe Long, could it be renamed as Long for Java docs? Guess it's more idiomatic for Java users